robottelo/ui/base.py

```diff
@@ -453,6 +453,23 @@ def is_element_visible(self, locator):
         self.wait_for_ajax()
         return element.is_displayed()
 
+    def element_type(self, locator):
+        """Determine UI element type using locator tag
+
+        :param locator: The locator of the element
+        :return: Returns element type value
+        :rtype: str
+
+        """
+        element = self.wait_until_element(locator)
+        if element is None:
+            return None
+        element_type = element.tag_name.lower()
+        if (element_type == 'input' and
+                element.get_attribute('type') == 'checkbox'):
+            element_type = 'checkbox'
+        return element_type
```

Comment:
Usually, I use such constructions to avoid unnecessary increase of indentation level, but here you are right. Thanks